src/test/java/examples/MainExample.java

```diff
@@ -20,12 +21,15 @@ public static void main(String... args) throws IOException {
                 .setPenalizeNl(true)
 //                .setNProbs(10)
                 .setMirostat(InferenceParameters.MiroStat.V2)
-                .setAntiPrompt("\n");
-
-        String modelPath = "/run/media/konstantin/Seagate/models/llama2/llama-2-13b-chat/ggml-model-q4_0.gguf";
+                .setAntiPrompt("User:");
+        String modelName = System.getProperty("model.name");
```

Comment:
This enables us to run the examples via maven directly rather than have it hard coded.  For instance, you can run this with the codellama model like so:
`mvn exec:java -Dexec.mainClass="examples.MainExample" -Dmodel.home="/Users/cstella/llm/models" -Dmodel.name="codellama-13b.Q5_K_M.gguf"`

Not sure if you would like this documented in the Readme.